### DISCUSSION TITLE
Agile T's & C's - new tariff code...

### DISCUSSION THREAD

**Discussion by willenic** [BottlecapDave/HomeAssistant-OctopusEnergy#755] [General]
I assume everyone else who is on agile has woken to find their entities aren't updating this morning? The T's and C's have changed and I guess my Tariff will change code to reflect the new standing charges and the fact its moving to a 12 month fixed term contract.

My MPAN and API key haven't changed so I assume when the tariff is updated by Octopus my target sensors and cost entities will resolve. My accumulative consumption flat lined on yesterdays end value and my targets and current rates all went pop overninght.

**Comment by intsoftl** [comment]
Same with Tracker which also changed (version/rates to fixed term) at midnight today.    Rates (both yesterday and tomorrow still show values from yesterday) no usage data today.

**Comment by willenic** [comment]
Looks like Octopus' knickers got untwisted very shortly after I left for work! All entities restored.
